burnP3PlusFireSTARR-1.0.3/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package 
+    name="burnP3PlusFireSTARR" version="1.0.3"
```

Comment:
I would say to keep formatting the same as the other metadata (i.e., name and version each get their own line) - helps with readability